Firestore/core/src/firebase/firestore/local/local_store.cc

```diff
@@ -430,38 +427,20 @@ QueryData LocalStore::AllocateTarget(Target target) {
   // Sanity check to ensure that even when resuming a query it's not currently
   // active.
   TargetId target_id = query_data.target_id();
-  HARD_ASSERT(target_ids.find(target_id) == target_ids.end(),
+  HARD_ASSERT(target_ids_.find(target_id) == target_ids_.end(),
               "Tried to allocate an already allocated target: %s",
               target.ToString());
-  target_ids[target_id] = query_data;
+  target_ids_[target_id] = query_data;
   return query_data;
 }
-void LocalStore::ReleaseQuery(const core::Query& query) {
-  ReleaseTarget(query.ToTarget());
-}
 
-void LocalStore::ReleaseTarget(const Target& target) {
+void LocalStore::ReleaseTarget(TargetId target_id) {
   persistence_->Run("Release target", [&] {
-    absl::optional<QueryData> query_data = query_cache_->GetTarget(target);
-    HARD_ASSERT(query_data, "Tried to release nonexistent target: %s",
-                target.ToString());
-
-    TargetId target_id = query_data->target_id();
-
-    auto found = target_ids.find(target_id);
-    if (found != target_ids.end()) {
-      const QueryData& cached_query_data = found->second;
-
-      if (cached_query_data.snapshot_version() >
-          query_data->snapshot_version()) {
-        // If we've been avoiding persisting the resume_token (see
```

Comment:
Just to confirm: you're deleting this logic because it was deleted on the other platforms already and this just brings iOS into line, right?

Reply:
Correct.